File: src/models/CSVExport.ts
export class CSVFeature {
    constructor(
        public module: string,
        public functionality: string,
        public tag: string,
        public sbpk: string,
        public adpk: string,
        public trpk: string,
        public fragment: string,
        public standard: string,
        public professional: string,
        public enterprise: string
    ) {}
}


export class CSVExport {
    constructor(
        public name: string
    ) {}
    public features: CSVFeature[] = [];
}

File: Models/CSVExport.cs
 namespace Configurator.Models
 {
     public class CSVFeature
    {
        public string Module { get; set; }
        public string Functionality { get; set; }
        public string Tag {get; set;}
        public string Sbpk {get; set;}
        public string Adpk {get; set;}
        public string Trpk {get; set;}
        public string Fragment {get; set;}
        public string Base {get; set;}
        public string Professional {get; set;}
        public string Enterprise {get; set;}
    }

    public class CSVExport
    {
        public string Name { get; set; }
        public CSVFeature[] Features { get; set; }
    }
 }
 
